src/analysis/souffle/examples/vision_pipeline/VisionPipelineTagOwners.dl

```diff
@@ -13,19 +13,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-----------------------------------------------------------------------------
-#include "authorization_logic.dl"
+#include "../../authorization_logic.dl"
```

Comment:
Using `..` in our include paths is not ideal. We should probably set up the `#include` roots in our invocation to souffle so that no prefix is needed or so that we use a full path from the root of the repo. I don't think this needs to be fixed right now, but something we should keep in mind once we get these things running automatically.

Reply:
OK I'll keep this in mind. I don't have a build script for this. I was just running souffle on it manually.